.pre-commit-config.yaml

```diff
@@ -38,11 +38,19 @@ repos:
         additional_dependencies:
           - mdformat-mkdocs==4.0.0
           - mdformat-footnote==0.1.1
+        exclude: |
+          (?x)^(
+            | docs/reference/\w+\.md
+          )$
```

Comment:
should we exclude these from all pre-commit hooks if they're generated files? If so, I'd add them to the global `exclude` regex at the top of this file rather than hook-specific `exclude` regexes

Reply:
I just copy pasted from Ruff's config

Reply:
Oh, that's how it was passing before... these were in the top-level one

Reply:
Thanks!